pkg/logrus.py

```diff
@@ -0,0 +1,45 @@
+"""
+Logger for orion
+"""
+
+import logging
+import sys
+
+
+class SingletonLogger:
```

Comment:
Can we move this `SingletonLogger` to `py-commons` and re-use it for both fmatch and orion?. Because logger implementation here is thread safe but not [here](https://github.com/cloud-bulldozer/py-commons/pull/22/files#diff-b9cafd7275d0e073f28114bdc785411940d553775ddb4cf59e04524a104b0a10R29)

Reply:
This makes sense.